src/lint.ts

```diff
@@ -99,12 +99,77 @@ export const lintAction = async (
       await bulkCompare(
         path.join(path.relative(topDir, vervetConfDir), resource.path),
         base,
+        undefined,
         resource.excludes,
       );
     }
   }
 };
 
+export const lintPRAction = async (
+  from?: string,
+  to?: string,
+): Promise<void> => {
+  console.log("Using Vervet API project configuration:");
+  const vervetConfDir = await findParentDirAsync(
+    path.resolve(process.cwd()),
+    ".vervet.yaml",
+  );
+  if (!vervetConfDir) {
+    console.log("no vervet conf");
+    throw new Error(
+      "cannot find .vervet.yaml -- is this a Vervet-managed API project?",
+    );
+  }
+  const vervetConfFile = path.join(vervetConfDir, ".vervet.yaml");
+  const vervetConfContents = await fs.readFile(vervetConfFile);
+  const vervetConf = loadYaml(vervetConfContents.toString()) as VervetConfig;
+
+  const topDir = await findParentDirAsync(vervetConfDir, ".git");
+  if (!topDir) {
+    console.log("not in a git repository");
+    throw new Error("cannot find .git -- is this a cloned git repository?");
+  }
+  process.chdir(topDir);
+
+  for (const [apiKey, apiValue] of Object.entries(vervetConf.apis)) {
+    console.log(`Linting API ${apiKey}`);
+    for (const resource of apiValue.resources) {
+      if (!resource.linter) {
+        console.log(`skipping API ${apiKey}: no linter`);
+        continue;
+      }
+      const linter = vervetConf.linters[resource.linter];
+      if (!linter || !linter["optic-ci"]) {
+        console.log(`skipping API ${apiKey}: not linted with optic-ci`);
+        continue;
+      }
+      const base = from ?? linter["optic-ci"]?.original ?? defaultBranchName;
+
+      await expectGitBranch(base);
+      await bulkCompare(
+        path.join(path.relative(topDir, vervetConfDir), resource.path),
+        base,
+        to,
+        resource.excludes,
+      );
+    }
+  }
+};
```

Comment:
The reason I created the second command is because we were using unnamed parameters and there is some very specific logic dependant on the first param: "resource". 
This conflicts with what we're trying to do in the PR workflow, where resource param is not used.

Do you know if there is a way to configure commander to accept something like:  sweater-comb lint base=sha1 to=sha2 and not rely on order?

Reply:
I refactored the code to reuse the existing lint command